src/microbe_stage/Microbe.cs

```diff
@@ -1975,6 +2006,36 @@ private void ProcessPhysicsForEngulfing()
         }
     }
 
+    private void HandleSprint(float delta)
+    {
+        if (isSprinting)
+        {
+            // Drain atp
+            var cost = BaseMovementCost(delta) * (1.0f + stress);
+            if (Compounds.GetCompoundAmount(atp) < Compounds.Capacity * Constants.MIN_ATP_TO_CONTINUE_SPRINT)
+            {
+                isSprinting = false;
+            }
+            else
+            {
+                Compounds.TakeCompound(atp, cost);
+                MovementFactor *= Constants.SPRINTING_MOVEMENT_MULTIPLIER;
+            }
+        }
+    }
+
+    private void HandleStress(float delta)
+    {
+        if (isSprinting)
+        {
+            stress += delta * Constants.STRESS_INCREASE_RATE;
+        }
+        else
+        {
+            stress = Math.Max(stress - delta * Constants.STRESS_INCREASE_RATE, 0.0f);
```

Comment:
wouldn't it be better to have a seperate `SPRINT_STRESS_RELIEF_RATE`?

Reply:
My idea was that this was a universal stress increase rate for anything strenuous happening. Even if we don't want that to be the case in the future, I would argue it's still better to split the rates when the time comes, to avoid implying that there is some other non-sprint thing in the code that has this effect. Small potatoes though.